### PR TITLE
x11-misc/dmenu: Version bump (5.1)

### PR DESCRIPTION
**Pull request by zsoltiv**
Signed-off-by: Zsolt Vadasz <zsolt_vadasz@protonmail.com>

### CODE REVIEW

File: x11-misc/dmenu/dmenu-5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (gyakovlev) [inline x11-misc/dmenu/dmenu-5.1.ebuild:4]: 8 ?

File: x11-misc/dmenu/dmenu-5.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
```
Comment (gyakovlev) [inline x11-misc/dmenu/dmenu-5.1.ebuild:1]: update upper bound to current year

File: x11-misc/dmenu/dmenu-5.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+EAPI=7
+inherit savedconfig toolchain-funcs
+
+DESCRIPTION="a generic, highly customizable, and efficient menu for the X Window System"
+HOMEPAGE="https://tools.suckless.org/dmenu/"
+SRC_URI="https://dl.suckless.org/tools/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 ~arm arm64 ~hppa ppc ppc64 ~riscv x86"
```
Comment (gyakovlev) [inline x11-misc/dmenu/dmenu-5.1.ebuild:13]: you have to reset all keywords to ~ on bump

**Review by gyakovlev — CHANGES_REQUESTED** [review]

**Comment by gyakovlev** [comment]
seems if your first PR (yay).

here are some tips for newcomers:
checkout a branch before doing a pull request, you did from master to master - that will bite you later.
install and run `pkgcheck scan --commits` while on PR branch, it will catch a lot of small mistakes and will help you write better ebuilds and submit better PRs.

consider using `pkgdev` - it writes proper commit messages, updates keywords and copyright years for you automatically.
so instead of `git commit` you'd do `pkgdev commit`
try it out!

and check rest of the docs here: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide  a lot of useful info and links.

**Comment by zsoltiv** [comment]
Alright, thanks for the heads-up!

File: x11-misc/dmenu/dmenu-5.1.ebuild (hunk 1 above)
Comment (zsoltiv) [reply x11-misc/dmenu/dmenu-5.1.ebuild:4]: savedconfig.eclass doesn't seem to support EAPIs higher than 7

**Review by gyakovlev — APPROVED** [review]

**Comment by gyakovlev** [comment]
thanks! merged.

**Pull request closed without merging**